### PR TITLE
PolyOverZq `sample_uniform`

### PR DESCRIPTION
**Pull request by jnsiemer**
**Description**

This PR implements...
- [x] uniform sampling

for `PolyOverZq`.

**Testing**

- [x] I added basic working examples (possibly in doc-comment)
- [x] I added tests for large (pointer representation) values
- [x] I triggered all possible errors in my test in every possible way
- [x] I included tests for all reasonable edge cases

**Checklist:**

- [x] I have performed a self-review of my own code
  - [x] The code provides good readability and maintainability s.t. it fulfills best practices like talking code, modularity, ...
    - [x] The chosen implementation is not more complex than it has to be
  - [x] My code should work as intended and no side effects occur (e.g. memory leaks)
  - [x] The doc comments fit our style guide
  - [x] I have credited related sources if needed

### CODE REVIEW

File: src/integer_mod_q/poly_over_zq/sample/uniform.rs [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        traits::GetCoefficient,
+    };
+    use std::str::FromStr;
+
+    /// Checks whether the boundaries of the interval are kept for small moduli.
+    #[test]
+    fn boundaries_kept_small() {
+        let modulus = Z::from(17);
+        for _ in 0..32 {
+            let poly_zq = PolyOverZq::sample_uniform(1, &modulus).unwrap();
```
Comment (Marvin-Beckmann) [inline src/integer_mod_q/poly_over_zq/sample/uniform.rs:87]:
```suggestion
        let poly_zq = PolyOverZq::sample_uniform(32, &modulus).unwrap();
        for _ in 0..32 {
```

**Review by Marvin-Beckmann — APPROVED** [review]
nice
